src/Uno.UWP/Storage/Pickers/FileOpenPicker.iOS.cs

```diff
@@ -66,13 +66,22 @@ private UIViewController GetViewController(bool multiple, TaskCompletionSource<S
 					};
 
 				case PickerLocationId.PicturesLibrary when multiple is true && iOS14AndAbove is true:
-				case PickerLocationId.VideosLibrary when multiple is true && iOS14AndAbove is true:
-					var configuration = new PHPickerConfiguration
+					var imageConfiguration = new PHPickerConfiguration
 					{
 						Filter = PHPickerFilter.ImagesFilter,
-						SelectionLimit = multiple ? 0 : 1
+						SelectionLimit = 0
+					};
+					return new PHPickerViewController(imageConfiguration)
+					{
+						Delegate = new PhotoPickerDelegate(completionSource)
+					};
+				case PickerLocationId.VideosLibrary when multiple is true && iOS14AndAbove is true:
+					var videoConfiguration = new PHPickerConfiguration
+					{
+						Filter = PHPickerFilter.VideosFilter,
+						SelectionLimit = 0
 					};
-					return new PHPickerViewController(configuration)
+					return new PHPickerViewController(videoConfiguration)
```

Comment:
@ajpinedam do we need to change the manual test to account for this scenario?

Reply:
The simulators do not have videos in the library, so it's difficult to test this from the manual Test. I had to add videos to test this scenario manually. 

cc: @jeromelaban

Reply:
@ajpinedam the samples app is on testflight so it can be tested properly, given a sample that details what do to.

Reply:
True, I got those updates :D . I will update the manual test so that we can use the Sample App to test this behavior. :)

Reply:
I ran the manual test and realized we do not need to change it. It already covers this scenario. We need to run the app on iOS 17, where the issue was found.

![image](https://github.com/unoplatform/uno/assets/1900897/1888339e-03ef-4e98-bc83-9ee774a60d35)

Reply:
Updated the sample's description to inform about iOS 17.0.